bioio_ome_tiff/tests/writers/test_ome_tiff_writer.py

```diff
@@ -96,6 +97,15 @@ def test_ome_tiff_writer_no_meta(
             scene = tiff.series[0]
             assert scene.shape == expected_read_shape
             assert scene.pages.axes == expected_read_dim_order
+    open_resource.close()
```

Comment:
This is likely just ignorance on my part, but is there a reason it's necessary to close the resource in this test and not in the other one?